src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -85,7 +85,7 @@
            :customization-color :blue}]
          [quo/wallet-graph {:time-frame :empty}]
          [quo/wallet-ctas
-          {:send-action #(rf/dispatch [:open-modal :wallet-select-address])
+          {:send-action #(rf/dispatch [:open-modal :wallet-select-address (:address account)])
```

Comment:
should we make the params a map here?
```
:open-modal :wallet-select-address {:address (:address account)}
```

Reply:
Would be cleaner, agree. But BTW, wondering if we actually need to pass this in navigation params, or maybe would be a better idea to handle this in the app db with subs and events, because either way the user will be able to switch accounts while being inside the send flow

Reply:
yeah that's a fair point. We could also do as a follow up if it's too much effort here? 🤔

Reply:
I think it is not needed so I removed, we can create a follow up in case we need it later